package.json

```diff
@@ -58,6 +58,6 @@
     "typescript": "^4.6.2"
   },
   "peerDependencies": {
-    "electron": "^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0 || ^14.0.0 || ^15.0.0 || ^16.0.0 || ^17.0.0 || ^18.0.0 || ^19.0.0"
+    "electron": "^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0 || ^14.0.0 || ^15.0.0 || ^16.0.0 || ^17.0.0 || ^18.0.0 || ^19.0.0" || ^20.0.0"
```

Comment:
Can you update the README.md, and also in one go update devDependencies to v20? Thanks!